components/raftstore/src/store/peer.rs

```diff
@@ -3115,6 +3115,41 @@ fn make_transfer_leader_response() -> RaftCmdResponse {
     resp
 }
 
+/// A poor version of `Peer` to avoid port generic variables everywhere.
+pub trait PlainPeer {
```

Comment:
What does `Plain` mean?

Reply:
For the context do you have some suggestions?